www/js/files.js

```diff
@@ -57,6 +57,20 @@
 
             },
 
+            deleteURIs: function(uris) {
+                console.log("deleteURIs", uris);
+                var deferred = null;
```

Comment:
If you put `var deferred = Promise.resolve();` here (think that's right?), then you wouldn't need the `if` inside the loop.